assets/app.js

```diff
@@ -1,11 +1,36 @@
 /* global hoodie */
+/**
+ * Hoodie is global to the project as it's loaded in the page before hand.
+ * Try executing 'window.hoodie' and 'hoodie' in your browser dev tools console.
+ * Look up Scope in JavaScript for more information
+ * http://www.smashingmagazine.com/2009/08/what-you-need-to-know-about-javascript-scope/
+ * Generally you want to avoid global variables in your applications as it will make your application unpredicatable.
+ */
```

Comment:
How about we remove this entire thing, including `/* global hoodie */`, this is not important for new developers, and experienced developers will already know. I wouldn't like for new developers to be this the first thing they see when they look into app.js